src/index.js

```diff
@@ -76,31 +77,41 @@ $(document).ready(function() {
         // projectName variable is defined in our example projects so the
         // correct test suite is automatically loaded. This sets default text
         // for <option> text and project indicator in top right corner.
+
         if (typeof projectName !== 'undefined') {
           projectNameLocal = projectName;
         }
 
+        let toggleElement = document.getElementById('toggle');
```

Comment:
I'm surprised this element has not been namespaced to `fcc_`. Not your problem, and I think it's a non-issue if we can put the UI in its own iframe. But if we cannot, this is something we should go back and fix. Maybe open an issue so we don't forget just in case?